about.html

```diff
@@ -0,0 +1,106 @@
+                <li class="active-link">
+                    <a class="nav-link" href="#">Drummers Work Shop</a>
+                </li>
+            </ul>
+        </nav>
+        <nav class="mobile-menu">
+            <div class="mobile-menu-bar" id="humburger-icon">
+                <div class="bar"></div>
+                <div class="bar"></div>
+                <div class="bar"></div>
+            </div>
+        </nav>
+    </header>
+
+    <section id="about-section">
+        <div class="about-heading">
+            <h3 id="about-intro">MUSIC FEST: DRUMMERS SUMMIT 2022</h3>
+            <p class="about-text">
+                 Drummers Summit 2022 is a initiative developed by world class drummers to promote drumming as a career.
+                Drummers Summit is usually held in every two years in a selected country of the world. This is to draw the attention
+                of drummers in selected country to the core skills and standards neccessary to build a successfull career as a drummer.
+                It provides a platform to connect with world class drummers around the world and build wider connections amongst drummers
+                <P class="about-footer">
+                    2022.10.26(WED) ~ 28(FRI)
+                </P>
+            </p>
+            <strong>Please register through the link <i>https://bit.ly/2233Bne</i> 2022 Drummers Summit</strong>
+        </div>
+    </section>
+
+    <section class="about-logo">
+        <h4 class="about-log-head">MUSIC FEST <br> DRUMMERS SUMMIT 2022 LOGO</h4>
+        <div class="program-bar"></div>
+        <div class="about-logo-text">
+            The logo is usually a local drum used the country which the summit would be taking place. It always
+            tries to have a cultural theme especially when the summit is holding in african countries.
+        </div>
+        <img class="about-logo-img" src="img/africa-logo.png" alt="Drummers Summit Logo">
+    </section>
+
+    <section class="past-events">
+        <h4 class="about-log-head">PAST DRUMMERS SUMMIT EVENTS</h4>
+        <div class="program-bar"></div>
+        <div class="about-logo-text">
+           Have a glance at some of the drummer summits held so far.
+        </div>
+        <div class="events">
+            <div class="event event_one">
+                <h4 class="event-year">2015</h4>
+                <p class="event-text">Drummers Summit in South Africa</p>
+            </div>
+            <div class="event event_two">
+                <h4 class="event-year">2017</h4>
+                <p class="event-text">Drummers Summit in Brooklyn</p>
+            </div>
+        </div>
+    </section>
+
+    <footer>
+        <div class="footer-logo">
+            <a href="home.html">
+                <img src="img/africa-logo.png" alt="">
+            </a>
+        </div>
+        <p class="footer-details">2022 Musical Fest Drummers Summit.</p>
+
+    </footer>
```

Comment:
**OPTIONAL** - The footer in your about page does not match the spec given in the design template. Please fix this issue 👍 👍. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project:
<img width="576" alt="about footer" src="https://user-images.githubusercontent.com/73607512/196936272-9730c469-7623-4e85-84ea-1853e9708295.PNG">

Design Template:
<img width="553" alt="template about footer" src="https://user-images.githubusercontent.com/73607512/196936435-bdfc778a-f761-4311-8647-dd8bcfd81781.PNG">